### PR TITLE
[TH] Translate pipe operator

### PR DESCRIPTION
**Pull request by iboss-ptk**
please review @zkan @lukkiddd

### CODE REVIEW

File: th/lessons/basics/pipe-operator.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  - /lessons/basics/pipe-operator/
+---
+
+pipe operator `|>` นั้นใช้สำหรับส่งผลลัพธ์ของ expression ไปเป็น argument ตัวแรกของ expression ถัดไป
+
+{% include toc.html %}
+
+## บทนำ
+
+การเรียก function ซ้อนกันเข้าไปหลายๆ ชั้น นั้นทำให้โปรแกรมเราดูรก และยากที่จะทำความเข้าใจ อย่างเช่นในตัวอย่างต่อไปนี้:
```
Comment (zkan) [inline th/lessons/basics/pipe-operator.md]: I think we can remove `:` at the end.

Comment (iboss-ptk) [reply th/lessons/basics/pipe-operator.md]: sure :D

**Comment by iboss-ptk** [comment]
@doomspork @nscyclone looks like I can not merge my own PR even if I approval from reviewer?

**Comment by doomspork** [comment]
Interesting.  @iboss-ptk what message/error do you get?  I can merge this for you until we figure it out.

**Comment by iboss-ptk** [comment]
@doomspork It just blocked by not having reviewer approval

**Comment by doomspork** [comment]
Oh that's tough.  For now @iboss-ptk you can ping me and I'll merge for you.

**Pull request merged**